server/controller/orders/updateOrderState.js

```diff
@@ -21,6 +23,9 @@ const updateOrderState = async (req, res, next) => {
     });
 
     const [order] = rows;
+    const userData = await getUserData({
+      userId: order.user_id,
+    });
```

Comment:
* hete you store the all result from query , destruct the rows to access user data 
```js
const { rows: result } = await getUserData({
      userId: order.user_id,
    });
    const [userData] = result;
```